tests/gateway/test_gateway.py

```diff
@@ -0,0 +1,28 @@
+from ddtrace.gateway import Addresses
```

Comment:
are the tests in the `gateway` directory run by a riot/tox target?

Reply:
I think I missed that! fixing!